plugins/net_plugin/net_plugin.cpp

```diff
@@ -789,6 +807,29 @@ namespace eosio {
             ( "_lport", local_endpoint_port );
          return mvo;
       }
+
+      // security group management
+      //
+   private:
+      std::optional<chain::account_name> participant_name_;
+      std::atomic<bool> participating_{false};
+
+   public:
+      /** @brief  Set/change the participant name for the connection
+       *
+       *  @param name   The name of the participant
+       */
+      void participant_name(chain::account_name name) { participant_name_ = name; }
```

Comment:
The TLS handling for this should be in the connection, so leave off the setter

Reply:
Removed setter